app/Actions/Shared/ComplexStatusCheck.php

```
@@ -113,8 +113,53 @@ private function aggregateContainerStatuses($application, $containers)
             }
         }
 
+        // If all containers are excluded, calculate status from excluded containers
+        // but mark it with :excluded to indicate monitoring is disabled
         if ($relevantContainerCount === 0) {
-            return 'running:healthy';
+            $excludedHasRunning = false;
+            $excludedHasRestarting = false;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $excludedHasRestarting. Keep variable name length under 20. (undefined)

(LongVariable)